src/content/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration.mdx

```diff
@@ -1469,18 +1348,9 @@ see: https://docs.newrelic.com/docs/apm/agents/ruby-agent/features/ruby-codestre
 
 </CollapserGroup>
 
+## Cross Application Tracer [#cross-application-tracer]
 
-## Errors inbox configuration [#errors-inbox-configuration]
-
-Setting one of the following tags will help you identify which versions of your software are producing the errors. 
-
-* `NEW_RELIC_METADATA_SERVICE_VERSION` will create tags.service.version on event data containing the version of your code that is deployed, in many cases a semantic version such as 1.2.3, but not always.
-* `NEW_RELIC_METADATA_RELEASE_TAG ` will create tags.releaseTag on event data containing the release tag (such as v0.1.209 or release-209).  
-* `NEW_RELIC_METADATA_COMMIT` will create tags.commit on event data containing the commit sha. The entire sha can be used or just the first seven characters (e.g., 734713b). 
-
-An upcoming release of errors inbox will automatically track which versions of your software are producing errors. Any version data will also be displayed in [CodeStream](/docs/codestream/how-use-codestream/performance-monitoring/#buildsha). 
```

Comment:
We don't have these environment variables/configuration options in the Ruby agent. If this is something that needs to appear in this document, [please open an issue in newrelic/newrelic-ruby-agent.](https://github.com/newrelic/newrelic-ruby-agent/issues/new?assignees=&labels=&projects=&template=story.md&title=)

Reply:
@meda123, do you know why this was added initially?